srtcore/queue.cpp

```diff
@@ -1255,7 +1235,7 @@ static string PacketInfo(const CPacket &pkt)
 }
 #endif
 
-EReadStatus CRcvQueue::worker_RetrieveUnit(ref_t<int32_t> r_id, ref_t<CUnit *> r_unit, sockaddr *addr)
+EReadStatus CRcvQueue::worker_RetrieveUnit(ref_t<int32_t> r_id, ref_t<CUnit*> r_unit, ref_t<sockaddr_any> r_addr)
```

Comment:
`ref_t` will be replaced globally soon with a dedicated PR. Until there it should stay because purposes for PRs should not be mixed.

I'll change this here to a plain reference as we already have an agreed convention for this, but only for the parameter that is changed here, so that the previous convention remains.